src/main/markdown/gettingstarted-outdated.md

```diff
@@ -0,0 +1,162 @@
+Getting Started
```

Comment:
[Cool URIs don't change](https://www.w3.org/Provider/Style/URI) - If we're moving the old content to a new page, we should link it from the "current" one, otherwise this page isn't going to be easy to find for those who could need it. On the other hand, if we think we should just entirely remove it, let's remove it, rather than keeping it but hard to find.

I don't think it would be unreasonable to keep the old contents on the existing page either, but all the way at the bottom with a "deprecated" header (so that old anchors could still work)? 

I'm not sure if I have a preference from the three choices... probably "move to new page, but link it properly, with a plan to delete in the future", but only a very mild preference.

Reply:
Yes, you are right. Missed it.